src/client/java/io/wispforest/limelight/api/entry/ResultEntry.java

```diff
@@ -24,4 +26,15 @@ public sealed interface ResultEntry permits InvokeResultEntry, SetSearchTextEntr
      * @return the text of this entry
      */
     Text text();
+
+    /**
+     * @return the prefix used to identify the extension in the search result.
+     * @apiNote By default, it'll use the translation if it exists, if it doesn't, it'll use the extension's name
+     * */
+    default Text prefix() {
+        String key = Util.createTranslationKey("limelightExtension", extension().id()) + ".prefix";
```

Comment:
tbh it feels like there should be a common method for getting the `limelightExtension.namespace.id` part

Reply:
like a "LimelightExtention.translationKey()"?

Reply:
`baseTranslationKey` probably, but yeah

Reply:
Sorry for the delay, github didn't update the page with your reply